grails-app/controllers/org/pih/warehouse/api/ProductApiController.groovy

```diff
@@ -56,7 +93,7 @@ class ProductApiController extends BaseDomainApiController {
         render([data: data] as JSON)
     }
 
-    def list = {
+    def search = {
```

Comment:
I think this might be a breaking change. The product API is already being used externally for things like the mobile app and other API clients so this might be a bad idea. In the future, please don't change existing endpoints unless we've had a discussion.